spec/fixtures/rightLicenses/rightLic1.js

```diff
@@ -0,0 +1,63 @@
+
```

Comment:
I would probably name the files accordingly, ie. `valid-license-spaces-before.js`, `valid-license-no-spaces-after.js`, `valid-license-shebang.js`. The same is for invalid license sample files.